varbase.install

```diff
@@ -459,3 +459,13 @@ function varbase_update_8038() {
     \Drupal::service('module_installer')->install(['entity_clone'], FALSE);
   }
 }
+
```

Comment:
This Update show be in Varbase Bootstrap Paragraphs

Reply:
But it's required by varbase landing page.

Reply:
Agree with you, Please move the update function to varbase_landing.
and it should be enabled only from varbase_landing.info.yml too for new installed websites.